service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -199,6 +204,56 @@ void resetInvocations() {
         }
     }
 
+    @Test
+    void shouldSendFailedPaymentNotificationOnPaymentsApiException() throws NotificationClientException {
+        given(ldClient.boolVariation(eq("payments"), any(), anyBoolean())).willReturn(true);
+        CaseDetails caseDetails = enableSendToCtscOnCaseDetails(YES);
+        caseDetails.getData().put("displayAmountToPay", YES.getValue());
+
+        doThrow(new PaymentsApiException(1, "", new Throwable())).when(paymentService)
+            .makePaymentForCaseOrders(any(), any());
+
+        postSubmittedEvent(caseDetails);
+
+        verify(notificationClient).sendEmail(
+            APPLICATION_PBA_PAYMENT_FAILED_TEMPLATE_FOR_LA,
+            "local-authority@local-authority.com",
+            Map.of("applicationType", "C110a"),
+            "12345");
+
+        verify(notificationClient).sendEmail(
+            APPLICATION_PBA_PAYMENT_FAILED_TEMPLATE_FOR_CTSC,
+            "FamilyPublicLaw+ctsc@gmail.com",
+            expectedCtscNotificationParameters(),
+            "12345");
+    }
+
+    @Test
+    void shouldSendFailedPaymentNotificationOnHiddenDisplayAmountToPay() throws NotificationClientException {
```

Comment:
I think I'd be good to have one more test for `displayAmountToPay` not being set on case data, verifying that no PBA failed notification was sent.

Reply:
good idea will add :)